app/containers/HomePage.js

```diff
@@ -8,7 +8,6 @@ import { connect } from 'react-redux';
 import * as HomeActions from '../actions/homePageActions';
 import Home from '../components/home/Home';
 
-
```

Comment:
A convention of the project is to use two new lines after imports. I have to make an eslint rule that enforces this.